ravenframework/Models/Dummy.py

```diff
@@ -152,9 +150,6 @@ def createNewInput(self,myInput,samplerType,**kwargs):
            a mandatory key is the sampledVars'that contains a dictionary {'name variable':value}
       @ Out, ([(inputDict)],copy.deepcopy(kwargs)), tuple, return the new input in a tuple form
     """
-    if len(myInput)>1:
-      self.raiseAnError(IOError,'Only one input is accepted by the model type '+self.type+' with name'+self.name)
-
```

Comment:
@mandd I guess we can remove this constraint for external model, but I think we still need it for ROM. Could you just remove this constraint in the external model.

Reply:
Fixed